File: Daily_code_61/Daily_code_61.py
def implementation(x,y):
    value = 1
    if y > 0:
        for i in range(0, y):
            value = value * x
        return value
    elif y == 0 :
        return value
    else:
        for i in range(0, y,-1):
            value = value * 1/2
        return value
    
x = 10
y = 2
answer = implementation(x,y)
print(answer)

File: Daily_code_61/Daily_code_61.js
function implementation(x, y){
    let value = 1;
    if(y > 0){
        for(let i = 0 ; i < y ; i++){
            value = value * x;
        }
        return value;
    }else if (y == 0) {
        return value;
    } else {
        for(let i = 0 ; i > y ; i--){
            value = value * 1/2;
        }
        return value;
    }
    
}

const x = 2;
const y = -3;
const answer = implementation(x,y);
console.log(answer);